mflowgen/full_chip/construct.py

```diff
@@ -54,6 +54,7 @@ def construct():
     # LVS
     'lvs_hcells_file'   : 'inputs/adk/hcells.inc',
     'lvs_connect_names' : '"VDD VSS VDDPST"',
+    'lvs_verify_netlist' : 0,
```

Comment:
What does this do?

Reply:
This stops LVS from complaining that dragonphy_top has missing dependencies. It seems like even when you do LVS BOX, the verify step tries to fully resolve the dependencies. The actual LVS works just fine.